code/modules/mob/holder.dm

```diff
@@ -64,31 +64,17 @@
 	desc = "It's a small maintenance robot."
 	icon_state = "drone"
 
-/obj/item/holder/Runtime
```

Comment:
These are things that can be kept in, with tweaks to make them use appropriate sprites.

Reply:
Readded